michaelnyu/magna
# New user entry

Established changes for the User Entry model. 

**Changed Files:** 
-Models.py: Updated with region field and a function to return the region
-Serializer.py: Updated to reflect models.py
-Views.py: Updated with new function to return all users for a specified region
-Urls.py: Updated to route user to new function in views.py


**Testing**
Steps to replicate views test: 

1. python3 manage.py runserver
2. Navigate to url. Append "/api/v1/(REGION-NAME)" 
3. See list of users of that region or 404 NOT FOUND


<img width="1280" alt="creativelabsproject" src="https://user-images.githubusercontent.com/30357600/36893111-b9a80058-1dbb-11e8-8681-c502bea87373.png">
